docs/azure_arc_jumpstart/azure_arc_ml/aks/aks_blob_mnist_arm_template/_index.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "Image Classification on MNIST dataset from Blob Storage"
+linkTitle: "Image Classification on MNIST dataset from Blob Storage"
+weight: 1
+description: >
+---
+
+## Train, Deploy and call inference on an image classification model - MNIST dataset from Azure Blob Storage
+
+The following README will guide you on how to deploy an end-to-end Machine Learning pipeline using [Azure Arc-enabled machine learning](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-attach-arc-kubernetes) deployed on [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) cluster using [Azure ARM Templates](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview). 
+
+By the end of this guide, you will have:
+* An AKS cluster onboarded with Azure Arc
+* A Microsoft Windows Server 2019 (Datacenter) Azure client VM (with optional steps outlined to remote access via [Azure Bastion](https://docs.microsoft.com/en-us/azure/bastion/bastion-overview) for increased Security Posture)
+* Azure Machine Learning with:
+  * Attached [Kubernetes (Preview) Compute](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-attach-compute-targets#kubernetes) (Azure Arc)
+  * Model Training Run performed on Kubernetes Cluster - using [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/ml/job?view=azure-cli-latest#az_ml_job_create)
+  * [Model Registered](https://docs.microsoft.com/en-us/azure/machine-learning/concept-model-management-and-deployment#register-and-track-ml-models) on Azure ML Studio (via [`pickle`](https://docs.python.org/3/library/pickle.html) file)
+  * Model Inference endpoint (pod) deployed on Kubernetes Cluster - using [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/ml/endpoint?view=azure-cli-latest#az_ml_endpoint_create)
+  * Model Inference endpoint (pod) invoked using [1. Azure CLI](https://docs.microsoft.com/en-us/cli/azure/ml/endpoint?view=azure-cli-latest#az_ml_endpoint_invoke) and [2. PowerShell native RESTful methods](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-restmethod?view=powershell-7.1). The second method is to demonstrate that the deployed model can also be called via [Postman](https://www.postman.com/) - or downstream business applications etc.
```

Comment:
Do we want to incl. Postman in the Client VM?

Reply:
I don't think it'll add much value, since we're already showing how to call the RESTAPI via PowerShell. I've removed Postman verbiage in the latest commit to avoid any confusions.